_i18n/it.yml

```diff
@@ -0,0 +1,97 @@
+langs:
+  en: English
+  es: Español
+  it: Italiano
+
+global:
+  date: '%Y/%m/%d'
+  monero: Monero
+  getting_started: Inizia da qui
+  copyright: Copyright
+  monero_project: The Monero Project
+  sitename: getmonero.org, The Monero Project
+  wiki: Moneropedia
+  tags: Articoli per Tag
+  wikimeta: su Moneropedia, l'enciclopedia su Monero
+  tagsmeta: Tutti gli articoli del blog su Monero che sono taggati
+  titlemeta: sulla home di Monero, la moneta digitale che è sicura, privata e intracciabile
+  terms: Termini
+  privacy: Privacy
+  copyright: Copyright
+  edit: Modifica questa pagina
+  untranslated: Questa pagina non è ancora stata tradotta. Se vuoi aiutare a tradurla vedi il
+titles:
+  index: Home
+  whatismonero: Cos'è Monero (XMR)?
+  using: Usare Monero
+  accepting: Accettare Monero
+  contributing: Migliorare Monero
+  mining: Minare Monero
+  faq: FAQ
+  downloads: Downloads
+  allposts: Tutti i post del blog
+  team: Monero Team
+  hangouts: Canali di comunicazione
+  events: Eventi
+  sponsorships: Sponsorizzazioni
+  merchants: Commercianti & Servizi
+  about: Riguardo Monero
+  roadmap: Roadmap
+  researchlab: Monero Research Lab
+  moneropedia: Moneropedia
+  userguides: Guide per Utenti
+  developerguides: Guide per Sviluppatori
+  technicalspecs: Specifiche Tecniche
+  themoneroproject: The Monero Project
+  presskit: Monero Press Kit
+  legal: Legale
+  ffs: Forum Funding System
+  ffs-cp: Completed Proposals
+  ffs-fr: Fondi Richiesti
+  ffs-ideas: Ideee
+  ffs-ot: Tasks aperte
+  ffs-wip: Work in Progress
+  blogbytag: Blog per Tag
+index:
+  page_title: "Monero - sicuro, privato, intracciabile"
+  what_is_1: Cos'è
+  what_is_2: " ?"
+  what_is_orange_block: "Monero è una valuta open-source sicura, privata, intracciabile e liberamente utilizzabile da chiunque."

```

Comment:
'intracciabile' non è una parola corretta in italiano, si può utilizzare 'non tracciabile' oppure 'non rintracciabile' oppure 'irrintracciabile'.